src/SFA.DAS.Forecasting.Application/AccountProjection/Services/AccountProjectionService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.Forecasting.Domain.AccountProjection;
+
+namespace SFA.DAS.Forecasting.Application.AccountProjection.Services
+{
+    public class AccountProjectionService : IAccountProjectionService
+    {
+        private readonly IAccountProjectionRepository _repository;
+
+        public AccountProjectionService(IAccountProjectionRepository repository)
+        {
+            _repository = repository;
+        }
+
+        public async Task<AccountProjectionExpiry> GetExpiringFunds(long expectedAccountId)
+        {
+            var projections = await _repository.GetAccountProjectionByAccountId(expectedAccountId);
+
+            var projectionForDate = projections.FirstOrDefault();
+            if (projectionForDate == null)
+            {
+                return null;
+            }
+
+            var projectionDate = projectionForDate.ProjectionCreationDate;
```

Comment:
Is there a reason why this var was created instead of just using projectionForDate.ProjectionCreationDate in the return statement?